docker/jenkins/docker-run.py

```diff
@@ -50,8 +50,8 @@
 # Execute the commands
 for cmd in config['cmd']:
     print('[+] ' + cmd)
-    execute = cli.exec_create(container['Id'], cmd=cmd, stdout=True, stderr=True)
-    for char in cli.exec_start(execute['Id'], tty=True, stream=True):
+    execute = cli.exec_create(container['Id'], cmd=cmd, stdout=True, stderr=True, stdin=False)
+    for char in cli.exec_start(execute['Id'], stream=True):
```

Comment:
I don't think `stdin` had anything to do with our issue (the default values was `False`) but removing `tty` which default to `False` did the trick.